src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java

```diff
@@ -266,18 +265,83 @@ public void testCacheDirectoryDoesNotExistAndShouldBeCreated() throws IOExceptio
     }
 
     @Test
-    public void testPathToCacheContainsOnlyFileName() throws IOException {
```

Comment:
Why is the original test removed? It seems it was testing a cache location that is only the file name and not any absolute path or sub-directory?

Reply:
reverted. done.